# Number of robots in the fleet at startup.
fleet_size: 4

# Publishes PoseStamped message for each robot to expand/shrink/load the swarm structure
tf_changer_topic: "tf_changer"

# Robot enable status topic.
robot_enable_status_topic: "robot_enable_status"

# Robot-level parameters.
# Frame topics for each robot. Controlling these controls the TF frame
# and not the physical robot.
robot_frame_command_topic: "/robot_frame_vel"

# Command topics for each robot. Controlling these controls the phyiscal robot.
robot_command_topic: "/d_/cmd_vel"

# Virtual (desired) robot (control point) frame names.
virtual_robot_frame_name: "tf_robot_"

# Actual robot (control point) frame names.
# Needed to be able to use the syncing desired robot frames function.
real_robot_frame_name: "d_tf_base_link"

# Robot desired state topic name.
# swarm_control.py publishes desired robot states here.
robot_desired_state_topic: "/d_/desired_state"

# Name of each robot. Used in the GUI.
robot_name: "D_"
